drone-fly-integration-tests/src/test/java/com/expediagroup/dataplatform/dronefly/core/integration/DroneFlyIntegrationTestUtils.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.dataplatform.dronefly.core.integration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import com.google.common.collect.Lists;
+
+public class DroneFlyIntegrationTestUtils {
+
+  static final String TOPIC = "apiary-events";
+  static final String DATABASE = "database";
+  static final String TABLE = "table";
+
+  public static Table buildTable() {
+    return buildTable(null);
```

Comment:
```suggestion
    return buildTable(TABLE);
```
Then you can get rid of the ternary operator on line 44 and assume that a valid table name is always passed in.

Reply:
Good idea. Done